cosmos/dbt/graph.py

```diff
@@ -52,6 +51,14 @@ class DbtNode:
     config: dict[str, Any] = field(default_factory=lambda: {})
     has_test: bool = False
 
+    @property
+    def name(self) -> str:
+        return self.unique_id.split(".", 2)[2].replace(".", "_")
```

Comment:
It would be great if we could link the documentation we're using as a reference to do this string parsing, and explain why we are doing this, as opposed to using the original `name`

It could be helpful to comment on the `unique_id` composition as well:
`<resource_type>.<package>.<resource_name>`

What about the replacement between "." and "_" - it would be great to document these.

We may want that this property use the other property, currently "select", since it is a subset of the logic.

Reply:
Updated.